ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -99,6 +99,20 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
         this.setState({showCompactNodes: !this.state.showCompactNodes});
     }
 
+    private getPageTitle(view: string) {
+        switch (view) {
+            case AppsDetailsViewKey.Tree:
+                return 'Application Details Tree';
+            case AppsDetailsViewKey.Network:
+                return 'Application Details Network';
+            case AppsDetailsViewKey.Pods:
+                return 'Application Details Pods';
+            case AppsDetailsViewKey.List:
+                return 'Application Details List';
+        }
```

Comment:
Maybe we can simply return a string chain like this?

```return `Application Details ${view[0].toUpperCase()}${view.slice(1)}`;```

Reply:
Good suggestion, however, if one day, the strings need to be pulled out and if we want translation, then we should keep them as is. Eg. we don't want the 'key' to be part of the displayed string.

Also, I'd like Alex's feedback on whether we should indeed change the title.  Originally, these views have "Application Details" as their title.  Here, I'm suggesting new titles, and Alex might prefer to keep it the same.